x/bank/types/msgs.go

```diff
@@ -79,6 +79,10 @@ func (msg MsgMultiSend) ValidateBasic() error {
 		return ErrNoInputs
 	}
 
+	if len(msg.Inputs) != 1 {
```

Comment:
let's also change the proto file for MsgMultiSend, in particular the `signer` proto option